src/pages/home/index.js

```diff
@@ -1,9 +1,10 @@
+/* eslint-disable no-unused-vars */
```

Comment:
If there are variables that are not being used is better to remove them instead of disabling the lint rule, WDYT?